ouroboros-consensus/src/Ouroboros/Consensus/Mempool/Impl.hs

```diff
@@ -290,516 +207,24 @@ forkSyncStateOnTipPointChange registry menv =
         , wReader      = getCurrentTip
         }
   where
+    MempoolEnv { mpEnvStateVar = istate
+               , mpEnvLedger = ldgr
+               , mpEnvTracer = trcr
+               , mpEnvLedgerCfg = cfg
+               , mpEnvCapacityOverride = co
+               } = menv
+
     action :: Point blk -> m ()
-    action _tipPoint = void $ implSyncWithLedger menv
+    action _tipPoint = void $ do
```

Comment:
Style: This layout is a little unconventional. Please keep to the rule that a `do` introduces a 2-space indent. (Also, please use `do $` instead of `do (`.

Reply:
True, fixed. (Resolving)